app/models/ahoy/event.rb

```diff
@@ -1,8 +1,24 @@
 class Ahoy::Event < ApplicationRecord
   include Ahoy::QueryMethods
 
+  SYSTEM_EVENTS = { 'comfy-blog-page-visit'=> 0, 'comfy-cms-page-update'=> 1, 'comfy-cms-page-visit'=> 2, 'comfy-file-update'=> 3, 'comfy-user-update'=> 4, 'email-visit'=> 5, 'forum-post-update'=> 6, 'forum-thread-visit'=> 7 }
```

Comment:
we should change this to the following, for clarity: 

``` ruby
SYSTEM_EVENTS = { 'comfy-blog-page-visit'=> 0, 'comfy-cms-page-update'=> 1, 'comfy-cms-page-visit'=> 2, 'comfy-cms-file-update'=> 3, 'subdomain-user-update'=> 4, 'subdomain-email-visit'=> 5, 'subdomain-forum-post-update'=> 6, 'subdomain-forum-thread-visit'=> 7 }
```

the idea is to prefix `comfy-cms` / `comfy-blog` for comfy related things (eg CMS, blog)
for other stuff, they should be prefixed under `subdomain` eg: `subdomain-email-visit`

Reply:
@alis-khadka :point_up:

Reply:
Shall we change such defined event names 
```ruby
ahoy.track(
        "comfy-user-update",
        {visit_id: current_visit.id, edited_user_id: @user.id, user_id: current_user.id}
      )
```
to ?
```ruby
ahoy.track(
        "subdomain-user-update",
        {visit_id: current_visit.id, edited_user_id: @user.id, user_id: current_user.id}
      )
```

Reply:
@restarone-solutions  CC @alis-khadka  

yes,  you will need to make that change. Otherwise things would be named incorrectly right? (as in, the two different event names would point at the same actual event)

Reply:
@alis-khadka to be clear: when you rename: 

1. `comfy-user-update` => `subdomain-user-update` -- update all references from `comfy-user-update` to `subdomain-user-update`
2. `comfy-file-update` => `comfy-cms-file-update` -- update all references from `comfy-file-update` to `comfy-cms-file-update`
... and so on

the bottom line is that _system defined names_ like `comfy-user-update` and keywords and should be stable and consistent

Reply:
```ruby
# Updating Modified Events
Ahoy::Event.where(name: 'comfy-file-update').update_all(name: 'comfy-cms-file-update')
Ahoy::Event.where(name: 'comfy-user-update').update_all(name: 'subdomain-user-update')
Ahoy::Event.where(name: 'email-visit').update_all(name: 'subdomain-email-visit')
Ahoy::Event.where(name: 'forum-post-update').update_all(name: 'subdomain-forum-post-update')
Ahoy::Event.where(name: 'forum-thread-visit').update_all(name: 'subdomain-forum-thread-visit')

# Deleting these events for the properties with: comfy_cms_page_id because we need to update properties by iterating
Ahoy::Event.where(name: 'comfy-cms-page-visit').destroy_all
```